Tools/tsan/suppressions_free_threading.txt

```diff
@@ -27,9 +27,6 @@ race:_PyObject_GC_SET_SHARED
 race:_PyObject_GC_TRACK
 race:_PyType_HasFeature
 race:assign_version_tag
-race:compare_unicode_unicode
```

Comment:
Is this related?

Reply:
There's not a fix for it, but I'm not seeing any races reported against it after disabling it (it's a helper function in `dictobject.c`)